barretenberg/cpp/src/barretenberg/goblin/CMakeLists.txt

```diff
@@ -1 +1 @@
-barretenberg_module(goblin stdlib_recursion ultra_honk eccvm translator_vm stdlib_sha256 stdlib_merkle_tree stdlib_primitives)
\ No newline at end of file
+barretenberg_module(goblin stdlib_recursion ultra_honk eccvm translator_vm stdlib_sha256 stdlib_merkle_tree stdlib_primitives client_ivc)
```

Comment:
Needing to add client_ivc to the goblin module is is again a symptom of having mock_circuits within goblin. There's a good but of restructuring to do here so I'd like to do a follow on to move things around.

Reply:
I don't understand why this is needed here. What's defined (not declared, but defined) in client_ivc but needed in goblin?